src/modules/missionHelper/i18n/en_US.json

```diff
@@ -43,12 +43,12 @@
 			"arff": "ARFF",
 			"battalion_chief_vehicles": "Battalion Chief Vehicle",
 			"boats": "Boat",
-			"brush_truck": "Wildland fire engines",
+			"brush_truck": "Wildland Fire Engines",
```

Comment:
I suggest to use both, singular and plural: "Wildland Fire Engine | Wildland Fire Engines" as it looks better at the end in missionHelper 🤓